contracts/handler/Venus/VenusAssetHandler.sol

```diff
@@ -8,15 +8,25 @@ import {IVenusComptroller, IVAIController, IPriceOracle} from "./IVenusComptroll
 import {FunctionParameters} from "../../FunctionParameters.sol";
 import {IThena} from "../../core/interfaces/IThena.sol";
 import {IAlgebraPool} from "@cryptoalgebra/integral-core/contracts/interfaces/IAlgebraPool.sol";
+import {ISwapHandler} from "../ISwapHandler.sol";
+import {Ownable} from "@openzeppelin/contracts-4.8.2/access/Ownable.sol";
 import "./ExponentialNoError.sol";
 
 /**
  * @title VenusAssetHandler
  * @notice This contract interacts with the Venus protocol to manage user assets and liquidity positions.
  * @dev Provides functions to get balances, handle markets, and process loans within the Venus protocol.
  */
-contract VenusAssetHandler is IAssetHandler, ExponentialNoError {
-  address immutable ROUTER_ADDRESS = 0x1b81D678ffb9C0263b24A97847620C99d213eB14;
+contract VenusAssetHandler is IAssetHandler, ExponentialNoError, Ownable {
+  ISwapHandler swapHandler;
+
+  constructor(address _swapHandler) {
+    swapHandler = ISwapHandler(_swapHandler);
+  }
+
+  function setNewSwapHandler(address _swapHandler) public onlyOwner {
```

Comment:
The user will still only be able to use one router. We can whitelist the allowed handlers and let the user pass the swap handler they want to use (best rate). There should be a whitelist check for the handler